<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiTrainer</class>
 <widget class="QMainWindow" name="UiTrainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/App.qrc">
    <normaloff>:/App.png</normaloff>:/App.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblTerm">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>lblTerm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblStatus">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>lblStatus</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblValue">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight: bold;  color: red;</string>
      </property>
      <property name="text">
       <string>lblValue</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actMain_Exit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actMain_RandomRow"/>
    <addaction name="actMain_Learn"/>
    <addaction name="actMain_Mark"/>
    <addaction name="actMain_Exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actMain_RandomRow">
   <property name="text">
    <string>Random row</string>
   </property>
   <property name="toolTip">
    <string>Random row</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actMain_Exit">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Exit.png</normaloff>:/Menu/Menu/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actMain_Learn">
   <property name="text">
    <string>Learn</string>
   </property>
   <property name="toolTip">
    <string>Learn</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actMain_Mark">
   <property name="text">
    <string>Mark</string>
   </property>
   <property name="toolTip">
    <string>Mark</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/App.qrc"/>
 </resources>
 <connections/>
</ui>
